pkg/cmd/admin/policy/policy.go

```diff
@@ -222,6 +238,22 @@ func (a ClusterRoleBindingAccessor) GetExistingRoleBindingNames() (*sets.String,
 	return ret, nil
 }
 
+func (a ClusterRoleBindingAccessor) GetRoleBinding(roleNamespace, roleBindingName string) (*authorizationapi.RoleBinding, error) {
```

Comment:
I guess you have to conform to the interface, but namespace should always be empty